index.js

```diff
@@ -0,0 +1,24 @@
+const header = document.querySelector('header');
+const logo = document.querySelector('.logo');
+const menuButton = document.querySelector('.menu-button');
+const hamburgerIcon = document.querySelector('.hamburger-icon');
+const xIcon = document.querySelector('.x-icon');
+const navMenu = document.querySelector('.nav-menu');
+
+menuButton.addEventListener('click', () => {
+  logo.classList.toggle('active');
+  hamburgerIcon.classList.toggle('active');
+  xIcon.classList.toggle('active');
+  menuButton.classList.toggle('active');
+  navMenu.classList.toggle('active');
+  header.classList.toggle('full-screen');
+});
```

Comment:
Currently, your mobile menu isn't closing after selecting an option. Please adjust this to follow the project requirements.

![req](https://user-images.githubusercontent.com/42799579/190258028-82a5a6c4-1ca1-4f08-bc20-50b3b36b7404.png)